packages/core/src/modules/vc/SignatureSuiteRegistry.ts

```diff
@@ -1,40 +1,27 @@
-import { KeyType } from '../../crypto'
+import type { KeyType } from '../../crypto'
+
 import { AriesFrameworkError } from '../../error'
+import { injectable, injectAll } from '../../plugins'
 
 import { suites } from './libraries/jsonld-signatures'
-import { Ed25519Signature2018 } from './signature-suites'
-import { BbsBlsSignature2020, BbsBlsSignatureProof2020 } from './signature-suites/bbs'
 
 const LinkedDataSignature = suites.LinkedDataSignature
 
+export const SignatureSuiteToken = Symbol('SignatureSuiteToken')
 export interface SuiteInfo {
   suiteClass: typeof LinkedDataSignature
```

Comment:
I would assume this is correct, but if is supposed to be generic between, for now, ed25519 and BBS+ then the `LinkedDataSignature` is not really a good name I think.

Reply:
I think the bbs suites also extend linked data signature.

Reply:
Nope it extends `LinkedDataProof`, and `LinkedDataSignature` also extends `LinkedDataProof`. 

@karimStekelenburg any reason you used `LinkedDataSignature` instead of `LinkedDataProof`?

Reply:
Hmm not sure tbh. Probably by accident